pubspec.yaml

```diff
@@ -36,11 +36,13 @@ flutter:
     - assets/camera/gallery.png 
     - assets/camera/video.png
     - assets/menu/
+    - assets/menu/svg/
     - assets/drawing/size1.png
```

Comment:
from next time, no need to itemize the assets. Just include the whole directory like:
  - assets/drawing/

Reply:
okay boss.